graspy/align/orthogonal_procrustes.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+from scipy.linalg import orthogonal_procrustes
+
+from .base import BaseAlign
+
+
+class OrthogonalProcrustes(BaseAlign):
+    """
+    Compute the matrix solution of the orthogonal Procrustes problem, which is
+    that given two matrices X and Y of equal shape, find an orthogonal matrix
+    that most closely maps X to Y.
```

Comment:
Thoughts on adding a new "Notes" section to make this part mathematically concrete?

Reply:
i added a couple sentences to the intro of the docstring. i think that it flows better than having a separate notes section, but if you want me to move the last two paragraphs to Note - i am happy to do that as well.

Reply:
@bdpedigo pondered it more and moved to Notes. predominantly to be consistent between using ticks for variables and ticks for math.